bf2fc6cc711aee1a0c2a/kas-fleet-manager
# fix(apiv1): add serviceaccounts and cloud_providers resources in ApiV1 endpoint

## Description


Add serviceaccounts and cloud_providers resources in ApiV1 endpoint. Also add the apiV1 endpoint in the OpenAPI spec


## Verification Steps


1. Run the swagger docs with `make run/docs` and go to `localhost` to make sure that the doc is properly rendered. 
2. Run the API locally with `make run` and then query the APIV1 endpoint to make sure that serviceaccount and cloud_providers resources are returned. 
```bash
curl --silent localhost:8000/api/managed-services-api/v1 | jq
{
  "id": "v1",
  "href": "/api/managed-services-api/v1",
  "kind": "APIVersion",
  "collections": [
    {
      "id": "kafkas",
      "href": "/api/managed-services-api/v1/kafkas",
      "kind": "KafkaList"
    },
    {
      "id": "serviceaccounts",
      "href": "/api/managed-services-api/v1/serviceaccounts",
      "kind": "ServiceAccountList"
    },
    {
      "id": "cloud_providers",
      "href": "/api/managed-services-api/v1/cloud_providers",
      "kind": "CloudProviderList"
    }
  ]
}
```
## Type of change


## Checklist (Definition of Done)
